engine/classes/Elgg/Export/Data.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Elgg\Export;
+
+use ArrayObject;
+use DateTime;
+
+/**
+ * Exported representation of an ElggData instance
+ *
+ * @property int $time_created
+ */
+abstract class Data extends ArrayObject {
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct($input = [], int $flags = ArrayObject::ARRAY_AS_PROPS, string $iterator_class = "ArrayIterator") {
```

Comment:
what is happening here? is this a github thingy?

Reply:
Yeah, it does that sometimes.

Reply:
It's a const from ArrayObject, https://secure.php.net/manual/en/class.arrayobject.php

Just a Github thing